### ISSUE TITLE
missing support for 'break' and 'increment/decrement'

### ISSUE THREAD

**Issue by ruemic** [mattmccray/liquid.js#43]
Hey, first off thanks for this super useful browser compatible port of Liquid!

I noticed these two features are missing:

Increment/Decrement Tag:
```
{% increment variable %}
{% decrement variable %}
```
See usage here: https://docs.shopify.com/themes/liquid-documentation/tags/variable-tags#increment

Break Tag: Exits a For Loop
https://github.com/Shopify/liquid/wiki/Liquid-for-Designers#tags
Example:
```
{% for product in products %}    
  {% if product.category contains 'beverages' %}
  {{ product.title }}  
  {% break %}
  {% endif %}
{% endfor %}
```

Has adding support for these been considered?

**Comment by mattmccray** [comment]
Yes, I'd like to bring liquid.js back up to parity with ruby Liquid -- Probably with version 2. (see #42)
